README.md

```diff
@@ -1,17 +1,62 @@
-# react-native-template-echobind
-
-Our Recommended template for React Native projects.
-
-## Included
-
-📓 Libraries:
-
-- React Native 0.57
-- TypeScript
-- react-navigation
-- Storybook 4
-- Detox for E2E tests
-- Emotion Native for styling
+# `react-native-template-echobind`
+
+<br />
+<p align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
+    <img src="https://camo.githubusercontent.com/d22763c73585cf5d4cf87534659689c2a6b3f214/68747470733a2f2f7265732d332e636c6f7564696e6172792e636f6d2f6372756e6368626173652d70726f64756374696f6e2f696d6167652f75706c6f61642f635f6c7061642c685f3235362c775f3235362c665f6175746f2c715f6175746f3a65636f2f76313439393437333135312f68326b3233696f6f3479687230676a746f636d792e6a7067" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">react-native-template-echobind</h3>
+
+  <p align="center">
+    Our Recommended template for React Native projects.
+    <br />
+    <br />
+    <a href="https://github.com/othneildrew/Best-README-Template">View Demo</a>
```

Comment:
should we remove this until we have a video or something else to link to?

Reply:
Oops...yes, I'll remove